impl/src/test/java/org/ehcache/impl/internal/store/heap/BaseOnHeapStoreTest.java

```diff
@@ -768,12 +758,7 @@ public void testComputeWhenExpireOnCreate() throws Exception {
       public String apply(String key, String value) {
         return "value";
       }
-    }, new NullaryFunction<Boolean>() {
-      @Override
-      public Boolean apply() {
-        return false;
-      }
-    });
+    }, () -> false);
```

Comment:
could be refactored in a method reference returning false

Reply:
Indeed, I guess my mistake is that this PR contains a few replacement of anonymous classes with lambdas instead of being just about the SPI classes removal.